cli/command/image/build.go

```diff
@@ -112,7 +112,11 @@ func NewBuildCommand(dockerCli command.Cli) *cobra.Command {
 	flags.VarP(&options.tags, "tag", "t", "Name and optionally a tag in the 'name:tag' format")
 	flags.Var(&options.buildArgs, "build-arg", "Set build-time variables")
 	flags.Var(options.ulimits, "ulimit", "Ulimit options")
-	flags.StringVarP(&options.dockerfileName, "file", "f", "", "Name of the Dockerfile (Default is 'PATH/Dockerfile')")
+
+	if &options.dockerfileName == " " {
+		&options.dockerfileName = '.'
```

Comment:
I think the default path is set in https://github.com/docker/cli/blob/master/cli/command/image/build/context.go#L31

Reply:
We think that there is no difference between in "docker build" with "docker build ." , if a path is not specified, it must be based on the same index. Context.go is still waiting for dockerfile.

Reply:
I think it is not default Dockerfile ' PATH ' , it is only default dockerfile name. I want to add the default ' Dockerfile Path ' .

Reply:
If you look at line 103, you can see that docker build currently requires at least one argument.